internal/testhelpers/e2e/session.go

```diff
@@ -81,7 +81,7 @@ func (s *Session) executablePath() string {
 	return exec
 }
 
-func New(t *testing.T, retainDirs bool) *Session {
+func New(t *testing.T, retainDirs bool, extraEnv ...string) *Session {
```

Comment:
I added functionality to define environment variables that are set during an entire session (for every `ConsoleProcess`)